content/tutorials/manage-the-rippled-server/install-rippled.md

```diff
@@ -46,59 +81,193 @@ This section assumes that you are using CentOS 7 or Red Hat Enterprise Linux 7.
 
-2. Install the Ripple RPM repository:
+That said, macOS is suitable for many development and testing tasks.
 
-        $ sudo rpm -Uvh https://mirrors.ripple.com/ripple-repo-el7.rpm
+This section assumes that you are using macOS X 10.0 or higher. ***TODO: okay?***
 
-3. Download the `rippled` software package:
+{% set n = cycler(* range(1,99)) %}
 
-        $ yumdownloader --enablerepo=ripple-stable --releasever=el7 rippled
+{{n.next()}}. Install [Xcode](https://developer.apple.com/download/).
 
-4. Verify the signature on the `rippled` software package:
+{{n.next()}}. Install Xcode command line tools. ***TODO: Comment from PM: Installing homebrew removes the need to install xcode command line tools separately. JHA: homebrew lists xcode command line tools as a requirement: https://docs.brew.sh/Installation#requirements. Do we keep or remove this step?***
 
-        $ sudo rpm --import https://mirrors.ripple.com/rpm/RPM-GPG-KEY-ripple-release && rpm -K rippled*.rpm
+        xcode-select --install
 
-5. Install the `rippled` software package:
+{{n.next()}}. Install [Homebrew](https://brew.sh/).
 
-        $ sudo alien -i --scripts rippled*.rpm && rm rippled*.rpm
+        ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 
-6. Configure the `rippled` service to start on system boot:
+{{n.next()}}. Update Homebrew.
 
-        $ sudo systemctl enable rippled.service
+        brew update
 
-7. Start the `rippled` service
+{{n.next()}}. Use Homebrew to install dependencies. ***TODO: Added boost per PM's request: Installing Boost directly is going to frustrate people - brew can do it in one command. I ended up installing Boost with Brew with no issues. Which release to choose? JHA: I did `brew install boost` and brew installs 1.67.0 in /usr/local/Cellar/boost/1.67.0_1 -- is this okay? If we install Boost via brew, do we no longer have to compile boost: ./bootstrap.sh and ./b2 cxxflags="-std=c++14"? Comment from PM: Build issues with Boost not finding lib (boost-context) in my case. JHA: when does this issue come up? How can we help the user avoid or address this issue?***
+
+        brew install git cmake pkg-config protobuf openssl ninja boost
+
+{{n.next()}}. Set the `BOOST_ROOT` environment variable to point to the new directory created by the Homebrew installation of Boost. Put this environment variable in your `.bash_profile` so it's automatically set when you log in. For example, add the following line to the file. To find your Boost install directory, use `brew info boost`. ***TODO: Feedback from PM: Have to deal with environment variables. “rc” files is a vague statement. JHA: If you do brew install boost - you still have to set env variables, correct? I did it using .bash_profile and removed references to "rc files" - is this okay? Or do we want to tell users that they can choose to put the env var in the .bashrc file instead of .bash_profile - is there any benefit to saying this?***
```

Comment:
If you install with homebrew you do not need to set your own environment variables. I think just mentioning its bash_profile would be fine. You can use your .bashrc, but it requires also using your bash_profile as far as I know. I just thought "rc files" could be misinterpreted.

Reply:
Got it. When I installed, it did not update my BOOST_ROOT env var, but that is probably because it was already set? Maybe it doesn't overwrite/update if a value exists? I've updated the text to address - let's review and see if it works for you.

Reply:
I'm not sure on this, but maybe you only need to set the BOOST_ROOT environment variable if you're compiling Boost yourself. If you're able to install a compatible version of Boost via Brew (per the debate above) then the part about setting your BOOST_ROOT env var may be moot.

Reply:
Sorry if I was not clear. You definitely must set the env variable if compiling yourself!